domain.yml

```diff
@@ -14,12 +14,15 @@ entities:
 slots:
   cuisine:
     type: text
-  location:
+  latitude:
+    type: text
```

Comment:
Check if slot type of float value works.

Reply:
ok